base/src/com/google/idea/blaze/base/ui/problems/BlazeProblemsView.java

```diff
@@ -46,16 +46,9 @@
 import org.jetbrains.annotations.NotNull;
 
 import javax.annotation.Nullable;
-import javax.swing.Icon;
+import javax.swing.*;
```

Comment:
drive-by: google style guide disallows wildcard imports

Reply:
Thanks! I'll fix that.
@hmemcpy do you know whether there is an intellij code-style settings I can use to make sure I don't violate it again? Because the next time I optimize imports, it will happen again..